python/tests/test_logistic_regression.py

```diff
@@ -441,6 +441,7 @@ def _func_test_classifier(
     cu_lr = cuLR(fit_intercept=fit_intercept, penalty=penalty, C=C, l1_ratio=l1_ratio)
     cu_lr.solver_model.penalty_normalized = False
     cu_lr.solver_model.lbfgs_memory = 10
+    cu_lr.solver_model.linesearch_max_iter = 20
```

Comment:
Good catch. How did you find this out.

Reply:
saw these params in the failing test and double checked against the code (remembered linesearch was changed recently).